integration_tests/index.d.ts

```diff
@@ -3,6 +3,7 @@ declare namespace Cypress {
     /**   * Custom command to signIn. Set failOnStatusCode to false if you expect and non 200 return code
      * @example cy.signIn({ failOnStatusCode: boolean })
      */
-    signIn<S = unknown>(options?: { failOnStatusCode: false }): Chainable<S>
+
+    signIn<S = unknown>(options?: { failOnStatusCode: false | true }): Chainable<S>
```

Comment:
Typecheck failing with type false set as we change this to true elsewhere (in integration_tests/support/commands.ts Not sure if this is the correct fix.

Reply:
This has been fixed in a separate ticket